start.sh

```diff
@@ -28,5 +31,9 @@ if [ "$(ls -A /root/repoFiles/)" ]; then
   rm -rf tmp/
 fi
 
+echo 'starting ssh server...'
 # -D flag avoids executing sshd as a daemon
-/usr/sbin/sshd -D
+/usr/sbin/sshd -D &
```

Comment:
Does `sshd` have output that we could piggy back off of? Are we certain that as soon as this line executes that the server is ready?

Reply:
Not that I could find. The closest is the `-d` argument for debug mode, which will make it log what it's doing, but it exits after the first connection.